tests/e2e/404.spec.js

```diff
@@ -0,0 +1,22 @@
+const { test, expect } = require('@playwright/test');
+
+test('check the 404 page', async ({ page }) => {
+  const response = await page.goto('/thispagereallywillnotexist');
+
+  // Check the page status.
+  expect(response.status()).toEqual(404);
+
+  // Check the page text.
+  const settingsText = await page.evaluate('window.p4bk_vars.page_text_404');
+  await expect(settingsText).toBeDefined();
+  const pageContent = await page.locator('.speech-bubble').innerHTML();
+  await expect(pageContent).toContain(settingsText.replace('\r\n', '\n'));
```

Comment:
This wasn't working without `replace('\r\n', '\n')`, but there might be a better way to do this 🤔

Reply:
Tested with international data, `.replace(/[\r]/g, '')` seems to work better

Reply:
Updated! 🙂